waterloop-site/src/components/PastGeeseTimeline/ImageCarousel.tsx

```diff
@@ -111,6 +148,14 @@ const ImageCarousel: React.FC = () => {
           </ArrowMobile>
         </MobileContainer>
       </DescriptionWrapper>
+      <Timeline>
+      <HorizontalLine />
+        <Increments>
+          {imgs.map((goose, i) => { 
+            return <IncrementText onClick={() => selectGoose(i)}>{goose.name.replace('Goose ','')}</IncrementText>;
```

Comment:
`return <IncrementText onClick={selectGoose(i)}>{goose.name.replace('Goose ','')}</IncrementText>;`

Reply:
Followed this, got an error but it might be due to the hook? Not sure.

![image](https://user-images.githubusercontent.com/71536798/123526647-4f1b3e00-d6a7-11eb-9dad-8bea1e55149a.png)